tox.ini

```diff
@@ -0,0 +1,32 @@
+[tox]
+envlist = flake8,yamllint,openapi
+skipsdist = True
+
+
+[testenv]
+passenv = PYTHONPATH
+
+
+[testenv:flake8]
+deps =
+    flake8
+    flake8-bugbear
+    flake8-import-order
+commands = flake8 src bin
+
+[flake8]
+max-line-length = 80
```

Comment:
I would set it to 120 - 80 sometimes is more boring than useful

Reply:
Sometimes... that's why I disabled E501 and enabled B950 instead (https://github.com/PyCQA/flake8-bugbear#opinionated-warnings).

Reply:
well we can start with the 10% tolerance then, and increase it if it becomes too messy